hyp3lib/rtc2color.py

```diff
@@ -130,6 +164,7 @@ def rtc2color(fullpolFile, crosspolFile, threshold, geotiff, cleanup=False,
   else:
     blue = (2.0*bp*(1 - blue_mask) + 5.0*zp*blue_mask)*255
   blue[blue==0] = 1
+  # FIXME: Is this the right mask (xp > 0)? Blue should be only copol and no crosspol
   blue = blue * mask
```

Comment:
> blue: simple bounce with very low volume scattering

This is attempting to mask our all no data and cleaned values, but only uses crosspol to define that. Blue should be mainly copol with very little crosspol. Does it make sense to mask based on crosspol only?

*Note: this is the main question of the user story*

Reply:
This is the right mask as long as the cleanup threshold is << than the user selected power threshold.